docs/docs/experimentation-analysis/experiment-time-series.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Experiment Time Series
+slug: /app/experiment-time-series
+---
+
+# Experiment Time Series
+
+Time series showing the cumulative results of your experiment though a given experiment update are available in-line in the experiment results.
+
+![Experiment Time Series](/images/experiment-time-series.png)
+
+Each data point in the time series represents the cumulative results of the experiment as of a historical experiment update. There will be as many data points as there have been experiment updates, although we do some coarsening to one result per day when there are multiple results per day and many days in the experiment.
```

Comment:
Suggestion for clarity:

Each data point represents cumulative results for that day. When there are multiple updates within a day, GrowthBook consolidates them into a single daily point to keep the chart easy to read.